Utils/Dataflow/README

```diff
@@ -133,8 +133,35 @@ following requirements:
 
 [3] EOP, End-of-Process marker. Default value: '\0' (NULL)
 
+==================
+* 3. Stage types *
+==================
+There are three types of stages:
+
+1. Source stage - an initial stage in a dataflow pipeline that gets data
+from an external source (CDS, Rucio, AMI, etc.).
+
+2. Transform stage - a transitional stage that performs single logical
+operation on the metadata. Set (chain) of transform stages forms the whole
+transformation process for messages produced by Source. Transformation
```

Comment:
The word "Source" should not be capitalized: "source".

Reply:
If it should go with lowercase initial letter, then I suggest to extend it to "source stage". Else one may take it as "metadata source" (just as in all the other cases where lowercase "source" appears), while what produces *messages* should clearly be the stage.

Reply:
@mgolosova, @Evildoor
I suppose to change it into a "source stage". :)